controllers/error-handling.js

```diff
@@ -0,0 +1,4 @@
+exports.handle500Errors = (error, request, response, next) => {
+  console.error(error);
+  response.status(500).send({ msg: "Internal Server Error" });
```

Comment:
You may also want to add your error handing for a 404 here in order to separate from app.js